lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -42,23 +84,41 @@ typedef _KeyDataResponseCallback = void Function(bool handled);
 class EnginePlatformDispatcher extends ui.PlatformDispatcher {
   /// Private constructor, since only dart:ui is supposed to create one of
   /// these.
-  EnginePlatformDispatcher._() {
+  EnginePlatformDispatcher() {
     _addBrightnessMediaQueryListener();
+    _addHighContrastMediaQueryListener();
```

Comment:
Suggestion: remove the method and call addListener directly.
```suggestion
    HighContrastSupport.instance.addListener(_updateHighContrast);
```